theme/js/slideshow.js

```diff
@@ -1,71 +1,67 @@
-addEventListener('load',ontheload());
-document.getElementById('fwbutton').onclick = fwimtog();
-document.getElementById('bwbutton').onclick = bwimtog();
+document.addEventListener('load', ontheload());
+
+var autoswitch = null;
 
 // imideatly switches slidess
-let slideIndex = 1;
+var slideIndex = 1;
 function toggle(fw) {
     let slides = document.querySelectorAll("#titelbild .field__item");
-    if (slides.length<=1){
+    if (slides.length <= 1) {
         return true
     }
     console.log(slides);
     console.log(slideIndex);
-    if(fw){
-        if (slideIndex == slides.length-1) {
+    if (fw) {
+        if (slideIndex == slides.length - 1) {
             slideIndex = 0;
         } else {
             slideIndex += 1;
         }
     } else {
         if (slideIndex == 0) {
-            slideIndex = slides.length-1;
+            slideIndex = slides.length - 1;
         } else {
             slideIndex -= 1;
         }
     }
-    for (let i=0;i < slides.length;i++) {
+    for (let i = 0; i < slides.length; i++) {
         if (i != slideIndex) {
-            for (let z=100;z > 0;z-=1) {
```

Comment:
this loop slows down the browser and breaks the slideshow

Reply:
so increase step size or solve it like you did with transition